portal/app/[locale]/ecosystem/page.tsx

```diff
@@ -8,12 +8,8 @@ import { useTranslations } from 'next-intl'
 import { DemoCard } from './_components/demoCard'
 import bitcoinKitImg from './_images/bitcoin_kit_large.png'
 import bitcoinKitIcon from './_images/bitcoin_kit_small.svg'
-import cryptoChordsImg from './_images/crypto_chords_large.png'
-import cryptoChordsIcon from './_images/crypto_chords_small.svg'
 import demosImg from './_images/demos_large.png'
 import demosIcon from './_images/demos_small.png'
-import hemiHatchlingsImg from './_images/hemi_hatchlings_large.png'
-import hemiHatchlingsIcon from './_images/hemi_hatchlings_small.png'
```

Comment:
I'm fairly sure these 4 images are not used elsewhere, so we can remove them too